modules/food-menu/food-item-detail.tsx

```diff
@@ -53,8 +53,7 @@ export class MenuItemDetailView extends React.Component<Props> {
 					</React.Fragment>
 				) : null}
 
-				{item.nutrition_details &&
-				Object.keys(item.nutrition_details).length > 1 ? (
+				{Object.keys(item.nutrition_details ?? {}).length > 1 ? (
```

Comment:
Is there a different way we can do this?

Reply:
Object.keys throws on a null input… I could use _.size (which essentially does this) if you'd like. 

That'd probably be nice. I'll do that.